### PR TITLE
Add release note for stash-v2024.12.18

### PR DESCRIPTION
**Pull request by anisurrahman75**

### CODE REVIEW

File: content/post/stash-v2024.12.18/index.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+- disaster-recovery
+- kubedump
+- kubernetes
+- mongodb
+- postgresql
+- restore
+- stash
+---
+
+We are pleased to announce the release of [Stash v2024.8.27](https://stash.run/docs/v2024.8.27/setup/), packed with new features. You can check out the full changelog [HERE](https://github.com/stashed/CHANGELOG/blob/master/releases/v2024.4.8/README.md). In this post, we'll highlight the changes done in this release.
```
Comment (ishtiaqhimel) [inline content/post/stash-v2024.12.18/index.md:19]: `[Stash v2024.12.18](https://stash.run/docs/v2024.12.18/setup/)`

`https://github.com/stashed/CHANGELOG/blob/master/releases/v2024.12.18/README.md`

**Review by ishtiaqhimel — CHANGES_REQUESTED** [review]

File: content/post/stash-v2024.12.18/index.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+### New Features
+
+#### Introducing the `multiDumpArgs` Parameter for MySQL Backups
+
+We are excited to introduce the `multiDumpArgs` parameter, which is available for all MySQL versions during the backup process.
+
+**What is `multiDumpArgs`?**
+
+The `multiDumpArgs` parameter can be specified under the `spec.task.params` section in the `BackupConfiguration`. It allows you to define multiple dump commands, which will be executed sequentially and separated by the Bash `&&` operator.
```
Comment (ishtiaqhimel) [inline content/post/stash-v2024.12.18/index.md:29]: It allows you to provide arguments for multiple dump commands, which will be executed sequentially.

**Review by ishtiaqhimel — CHANGES_REQUESTED** [review]

File: content/post/stash-v2024.12.18/index.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+We are excited to introduce the `multiDumpArgs` parameter, which is available for all MySQL versions during the backup process.
+
+**What is `multiDumpArgs`?**
+
+The `multiDumpArgs` parameter can be specified under the `spec.task.params` section in the `BackupConfiguration`. It allows you to provide arguments for multiple dump commands, which will be executed sequentially.
+
+**Why Use `multiDumpArgs`?**
+
+This feature is designed for use cases where you need to take database backups into different dump files, such as:
```
Comment (hmsayem) [inline content/post/stash-v2024.12.18/index.md:33]: do we create different dump files?

**Review by hmsayem — CHANGES_REQUESTED** [review]

File: content/post/stash-v2024.12.18/index.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+
+### New Features
+
+#### Introducing the `multiDumpArgs` Parameter for MySQL Backups
+
+We are excited to introduce the `multiDumpArgs` parameter, which is available for all MySQL versions during the backup process.
+
+**What is `multiDumpArgs`?**
+
+The `multiDumpArgs` parameter can be specified under the `spec.task.params` section in the `BackupConfiguration`. It allows you to provide arguments for multiple dump commands, which will be executed sequentially.
```
Comment (hmsayem) [inline content/post/stash-v2024.12.18/index.md:29]:
```suggestion
We are excited to introduce the `multiDumpArgs` parameter, which is available for all MySQL versions during the backup process. The `multiDumpArgs` parameter can be specified under the `spec.task.params` section in the `BackupConfiguration`.
```

File: content/post/stash-v2024.12.18/index.md [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+**Why Use `multiDumpArgs`?**
+
+This feature is designed for use cases where you need to take database backups into different dump files, such as:
+
+- Backing up `table-schema` and `table-data` into separate files.
+
+- Customizing multiple dump operations to suit your specific needs.
+
+Using `multiDumpArgs`, you have to provide dump arguments by separating each command with the `$args` placeholder.
+
```
Comment (hmsayem) [inline content/post/stash-v2024.12.18/index.md:40]: You can specify multiple dump arguments for different `mysqldump` commands by separating each with `$args` using the `multiDumpArgs` parameter. Here is an example:

**Review by hmsayem — CHANGES_REQUESTED** [review]

File: content/post/stash-v2024.12.18/index.md (hunk 3 above)
Comment (anisurrahman75) [reply content/post/stash-v2024.12.18/index.md:33]: Yes, It creates a distinct dump file for each dump command.

**Review by ishtiaqhimel — DISMISSED** [review]

**Pull request merged**